apps/comments-ui/test/e2e/actions.test.ts

```diff
@@ -200,21 +200,57 @@ test.describe('Actions', async () => {
 
         // Should indicate this was a reply to a reply
         await expect(frame.getByTestId('comment-in-reply-to')).toHaveText('This is a reply to 1');
+
+        return {frame};
     }
 
     test('Can reply to a reply', async ({page}) => {
         mockedApi.addComment({
+            id: '1',
             html: '<p>This is comment 1</p>',
             replies: [
                 mockedApi.buildReply({
-                    html: '<p>This is a reply to 1</p>'
+                    html: '<p>This is a reply to 1</p>',
+                    parent_id: '1'
                 })
             ]
         });
 
         await testReplyToReply(page);
     });
 
+    test('can highlight reply when clicking on reply to: message', async ({page}) => {
+        mockedApi.addComment({
+            html: '<p>This is comment 1</p>',
+            replies: [
+                mockedApi.buildReply({
+                    html: '<p>This is a reply to 1</p>'
+                })
+            ]
+        });
+
+        const {frame} = await testReplyToReply(page);
```

Comment:
Why is this needed here? We've already tested that you can reply to a reply in the tests above, I think we can just set up the data that we need in the mockedApi instance so it's easier to see what the expected state is for the test and avoid the extra/unnecessary expectations and waits.